omnibus/config/software/oc_id.rb

```diff
@@ -28,6 +28,18 @@
 build do
   env = with_standard_compiler_flags(with_embedded_path)
 
+  # Tell nokogiri to use the system libraries instead of compiling its own
+  env["NOKOGIRI_USE_SYSTEM_LIBRARIES"] = "true"
```

Comment:
I just referenced it from omnibus nokogiri changes, not sure why both are required, but there is note saying don't remove it. https://github.com/chef/omnibus-software/blob/master/config/software/nokogiri.rb#L34-L56